### PR TITLE
Provide jupyter lab with this toolkit

### LINKED ISSUE #7: Install jupyterlab together with python
#### Description:

This will provide a standard way for people to develop machine learning and numerical computation code.
Jupyter lab is an interface to python and to a lots of other things, included a Jupyter notebook.
Jupyter notebooks are the equivalent of the scripts (like Grasshopper files).

Check this to understand for JupyterLab can do:
https://towardsdatascience.com/jupyter-lab-evolution-of-the-jupyter-notebook-5297cacde6b

### PR DESCRIPTION
**Pull request by epignatelli**
### Issues addressed by this PR

Closes #7 

This pr is installing [jupyter lab](https://jupyterlab.readthedocs.io/en/stable/) together with this toolkit, as a unified way to develop python code.

### Test files

- Compile this toolkit
- Compile the Numpy toolkit
- Compile the BHoM_UI
- Open grasshopper and place a `JupyterLab` component on the canvas
- Switch it on and check if the server is running.

### CODE REVIEW

File: Python_Engine/Compute/JupyterLab.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2019, the respective contributors. All rights reserved.
```
Comment (FraserGreenroyd) [inline Python_Engine/Compute/JupyterLab.cs]: Copyright should be to 2020

File: Python_Engine/Compute/JupyterLab.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        public static void JupyterLab(int port = 7777, bool start = false)
+        {
+            string path = Path.Combine(Query.EmbeddedPythonHome(), "Scripts", "jupyter-lab.exe");
+            if (!File.Exists(path))
+                throw new FileNotFoundException("Cannot find jupyter lab", path);
+
+            path = Path.GetFullPath(path);
+            string configPath = Path.Combine(Query.EmbeddedPythonHome(), "Scripts", "jupyter_lab_config.py");
+            if (start)
+                RunCommandAsync($"cd ../../../../ && {path} --port {port} --config {configPath}", false);
```
Comment (FraserGreenroyd) [inline Python_Engine/Compute/JupyterLab.cs]: Is this relative path guaranteed to always work? Or is it better to let users specify a path with this as a default?

Comment (epignatelli) [reply Python_Engine/Compute/JupyterLab.cs]: The relative path will work on all Windows 10 machines for sure, but I like that! Just pushed!

**Comment by epignatelli** [comment]
hey @FraserGreenroyd would you mind updating your review?

**Comment by FraserGreenroyd** [comment]
> hey @FraserGreenroyd would you mind updating your review?

How soon do you want it? I haven't had chance to fully test the implementation yet but can see @alelom is happy? I'm on leave this week but could test tomorrow or Friday if it's urgent and you want my review - otherwise I'm happy for you to dismiss it if you're happy with @alelom review :smile:

**Comment by epignatelli** [comment]
Hey, no, no worries! Thanks @FraserGreenroyd !

**Pull request merged**